docs/Converters/UKBtoBIDS.md

```diff
@@ -1,70 +1,83 @@
+| T2 Flair    | T2_FLAIR.nii.gz | nifti |                                                                                                                                           Defaced and cropped so there is no neck. Same as T1.                                                                                                                                           |
+| DWI         | AP/PA.nii.gz     | nifti |                                                                                                                                          Rawest existing images with available bvals and bvec.                                                                                                                                           |
+| rsfMRI      | rsfMRI.dcm | dicom |                                                                                                               The nifti doesn't always include a json and in consideration for its usage, the dicom is converted instead.                                                                                                                |
+| tfMRI       | tfMRI.dcm   | dicom |                                                                                                                                                             Same as rsfMRI.                                                                                                                                                              |
+| SWI         | SWI.nii.gz      | nifti | Combined coil version. We would get a rawer version, but `dcm2niix` has trouble handling the slices direction, so it is simpler to go with this version. In addition, SWI modality is not fully integrated to BIDS specification and some changes may be coming. A simple version is available in the current version of this converter. |
+
+
+### Understanding the command line
 
 The converter can be run with the following command line:
 
-```Text
-clinica convert ukb-to-bids [OPTIONS] DATASET_DIRECTORY CLINICAL_DATA_DIRECTORY BIDS_DIRECTORY 
+```bash
+clinica convert ukb-to-bids DATASET_DIRECTORY CLINICAL_DATA_DIRECTORY BIDS_DIRECTORY [OPTIONS]
 ```
 
 where:
 
-- `DATASET_DIRECTORY` is the path to the original UK BIobank imaging directory, whose content should look like:
+<div class="grid" markdown>
+
+=== "Imaging data :"
+
+    - `DATASET_DIRECTORY` is the path to the
+    original UK BIobank imaging directory.
+
+    - `BIDS_DIRECTORY` is the path to the
+    output directory where the BIDS-converted
+    version of UK Biobank will be stored.
+
 
-    ```text
+```title="DATASET_DIRECTORY Organisation"
```

Comment:
I think this is great. Would it be possible to have the same example box for the clinical data as well ?
When looking at the rendered page it almost looks like it was supposed to be there but was not rendered.